benchmarks/backend_request_func.py

```diff
@@ -224,9 +224,6 @@ async def async_request_openai_completions(
     pbar: Optional[tqdm] = None,
 ) -> RequestFuncOutput:
     api_url = request_func_input.api_url
-    assert api_url.endswith(
```

Comment:
maybe instead of deleting it we add another condition for start_profile?

Reply:
I feel the naming of this function should be changed instead.